### PR TITLE
chore(formatting): Ensure code generated by default is formatted

### PR DESCRIPTION
**Pull request by tomciopp**
Handles second item in issue #2605. LMKWYT

### CODE REVIEW

**Comment by mikeover** [comment]
I believe that most, if not all, of this is already included in #3033

File: installer/templates/phx_web/views/error_helpers.ex [hunk 1]
```diff
@@ -9,8 +9,8 @@ defmodule <%= web_namespace %>.ErrorHelpers do
   Generates tag for inlined form input errors.
   """
   def error_tag(form, field) do
-    Enum.map(Keyword.get_values(form.errors, field), fn error ->
-      content_tag :span, translate_error(error), class: "help-block"
+    Enum.map(Keyword.get_values(form.errors, field), fn (error) ->
```
Comment (chrismccord) [inline installer/templates/phx_web/views/error_helpers.ex]: ✂️ on parens around (error)

File: installer/templates/phx_web/router.ex [hunk 2]
```diff
@@ -14,7 +14,8 @@ defmodule <%= web_namespace %>.Router do
   end<%= if html do %>
 
   scope "/", <%= web_namespace %> do
-    pipe_through :browser # Use the default browser stack
+    # Use the default browser stack
```
Comment (chrismccord) [inline installer/templates/phx_web/router.ex]: let's kill this comment as it's not useful anyway

**Comment by chrismccord** [comment]
I just saw your 2nd comment, you are right it is all already taken care of in #3033. Thanks!!!

**Pull request closed without merging**